components/layout/Layout.tsx

```diff
@@ -5,7 +5,6 @@ import React, { useState, useEffect, Fragment } from "react";
 import Head from "next/head";
 import Link from "next/link";
 import styled from "styled-components";
-import Footer from "./Footer";
```

Comment:
This file is the Main Layout for the whole app, if you remove the `Footer` component from this file it will not render on all the pages.